### PR TITLE
Knn predict

### PR DESCRIPTION
**Pull request by MSBradshaw**
Added test data to the load data module.
Cleaned up main.py

### CODE REVIEW

**Review by jdayton3 — APPROVED** [review]
Looks good!

File: main.py [hunk 1]
```diff
@@ -1,63 +1,24 @@
...
+#create test and train labels
+gender_labels = data.clinical['gender'].tolist()
+MSI_labels = data.clinical['msi'].tolist()
+test_gender_labels = data.test_clinical['gender'].tolist()
+test_MSI_labels = data.test_clinical['msi'].tolist()
 
-#create the labels for which samples have been mislabeled
-mismatch_labels = labels.mismatch.tolist()
+#train learners for gender and msi here:
+knn_gender = lf.train_knn(data.proteomic.iloc[:,1:],gender_labels)
```
Comment (jdayton3) [inline main.py:13]: `LoadData()` reads only the quantitative data in, so the `.iloc[:,1:]` may not be necessary?

Comment (jdayton3) [reply main.py:13]: The gene names & sample IDs become the column and row indices.

File: main.py [hunk 2]
```diff
@@ -1,63 +1,24 @@
...
-
-#train a learner
-knn_gender = lf.train_knn(protein,gender_labels)
-knn_msi = lf.train_knn(protein,MSI_labels)
-
-
-lf.generate_and_write_results(test_pro,knn_gender,knn_msi,test_gender_labels,test_MSI_labels,list(test_cli.index))
\ No newline at end of file
+#make final predictions here, give it the two trained classifiers
+lf.generate_and_write_results(data.test_proteomic.fillna(0.0).iloc[:,1:],
```
Comment (jdayton3) [inline main.py:19]: `LoadData`'s constructor calls `fillna(0.0)`, so we might be able to remove that here.

Comment (MSBradshaw) [reply main.py:19]: I know the fill fix_data() function should do the fillna(), but it doesn't for some unknown reason

File: main.py (hunk 1 above)
Comment (MSBradshaw) [reply main.py:13]: You are correct, good catch

**Pull request merged**